lib/models/extensions/payment_title_extension.dart

```diff
@@ -1,6 +1,9 @@
 extension PaymentTitleExtension on String {
   // TODO(erdemyerebasmaz): Remove once default descriptions from the SDK is removed
   bool get isDefaultDescription {
-    return this == 'Send to L-BTC address' || this == 'Liquid transfer' || this == 'Lightning payment';
+    return this == 'Send to L-BTC address' ||
+        this == 'Liquid transfer' ||
+        this == 'Lightning payment' ||
+        this == 'Pay to Misty Breez';
```

Comment:
Is it worth making this default offer description a constant now?

Reply:
Yes, applied: c5347cfc0d8973c24be387060d6062579dc0bdec